src/fireflyClient.ts

```diff
@@ -233,7 +233,7 @@ export class FireflyClient {
    * @returns Boolean true if user is funded, false otherwise
    */
   async mintTestUSDC(contract?: address): Promise<boolean> {
-    const tokenContract = this.getContract("USDTToken", contract);
+    const tokenContract = this.getContract(this._usdcToken, contract);
```

Comment:
Can we check over here that what is the network user is connected to? If it is dev or testnet, we should not mint test tokens.

Reply:
But on the function description above, we have written that it works on testnet only. Let me know on which environments it should work and i'll update the docs too

Reply:
this function will also work on devnet but I don't think normal user's will have access to that, only developers will know about it. Either you can check at the beginning of the function what network user is connected to, if its PRODUCTION, this function should throw an error.

Reply:
Network is the object containing RPC url, chain id etc, Currently, we only have two added TESTNET and DEV, so I'll just create a condition if Network is other than any of these two throw error